src/views/homepage/components/featured-content-grid/search-featured-card.module.css

```diff
@@ -56,6 +56,7 @@
 		transparent 60%,
 		#2a283a40 90%
 	);
+	border-radius: 10px;
```

Comment:
🗒️ Oddly, `border-radius` had to be added on these 2 child elements for Safari, because the `overflow: hidden` property on the parent wasn't clipping the corners. 🤔